hieradata/vagrant.yaml

```diff
@@ -29,19 +29,19 @@ govuk::apps::stagecraft::worker::enabled: true
 govuk::apps::stagecraft::beat::enabled: true
 govuk::apps::stagecraft::celerycam::enabled: true
 
-# An example configuration for env vars
 govuk_containers::apps::release::envvars:
-  - "GOVUK_ENV=production"
-  - "NODE_ENV=production"
-  - "RACK_ENV=production"
-  - "RAILS_ENV=production"
+  - "FOO=BAR"
+
+govuk_containers::app::config::global_envvars:
+  - "GOVUK_ENV=development"
```

Comment:
I think govuk_env defaults to 'production' in the deploy::config class, and we only override with 'development' in:
`hieradata/development.yaml:govuk::deploy::config::govuk_env: 'development'`